app/scripts/modules/core/pipeline/config/triggers/webhook/webhookTrigger.html

```diff
@@ -0,0 +1,32 @@
+<ng-form name="webhookForm">
+
+    <div class="row">
+      <div class="col-md-3 sm-label-right">
+        Webhook URL
+      </div>
+      <div class="col-md-6">
+        http://localhost:8089/webhooks/webhook</i>
```

Comment:
Probably best to not use localhost:8089 as the default value ... should be able to reference the `gateUrl` for a more realistic value.